ts/data/case.ts

```diff
@@ -191,7 +191,11 @@ export const caseDefinition: TableDefinition<Case, Test> = {
         );
         tooltipInstance.addTarget(
           tumourDetailDiv,
-          document.createTextNode("Timepoint")
+          document.createTextNode(
+            `Tissue Origin: ${kase.tissueOrigin}\n` +
+              `Tissue Type: ${kase.tissueType}` +
+              (kase.timepoint ? `\nTimepoint: ${kase.timepoint}` : "")
```

Comment:
These don't render as line breaks in the browser. The easiest way to achieve this is to put each line in its own `div` as they are block by default and will stack vertically